packages/pluggableWidgets/badge-native/tests/e2e/Badge.spec.ts

```diff
@@ -1,37 +1,50 @@
-import { by, device, element, expect, waitFor } from "detox";
-import { Badge } from "./Badge";
-import { Alert } from "./Alert";
+import { device, expect, waitFor } from "detox";
+import { Widget } from "../../../../../tests/e2e/helpers-native/Helpers";
```

Comment:
what about move those helper methods to the `piw-native-utils-internal`? Then the imports will not be that relative (../../../) anymore.

Reply:
Nice suggestion! Learnt something new and made changes as needed 👍